common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -238,7 +238,7 @@ def get_html(self):
         html_id = self.location.html_id()
         if getattr(settings, 'PERFORMANCE_GRAPHITE_URL', '') != '' and \
```

Comment:
we shouldn't reference 'PERFORMANCE_GRAPHITE_URL' since we are no longer using Graphite.